resources/js/Pages/Praktikan.vue

```diff
@@ -411,15 +411,17 @@
                             v-on:click="showNilaiTA = false">
                           <div class="h-full w-full flex font-merri text-xl bg-gray-800 rounded-lg text-center m-auto">
                             <div class="m-auto text-white">
-                              Lanjut Ke Presentasi
+                              Lanjut Ke Jurnal
                             </div>
                           </div>
                         </div>
                       </div>
                     </div>
                   </div>
                 </div>
-                <div class="w-1/4 h-full flex-row overflow-y-hidden">
+                <div class="w-1/4 h-full flex-row overflow-y-hidden"
```

Comment:
A bit forget about the frontend of practicum sequence, can you check later? @adhiiimf

Reply:
For the frontend of practicum sequence is:
1. Intro
![start](https://user-images.githubusercontent.com/88444330/188271757-a53a5cd3-fb99-492f-9b1e-7edf31c5c006.jpg)
2. TA start
![TA](https://user-images.githubusercontent.com/88444330/188271767-f50839ee-ba8a-4b1c-9a06-a2959d3ba4dc.jpg)
3. After TA (show the score)
![score TA](https://user-images.githubusercontent.com/88444330/188271765-2385893a-dcaf-4ccf-bb55-33e903b504cf.jpg)
4. Jurnal
![jurnal](https://user-images.githubusercontent.com/88444330/188271764-2f6afaa7-eb27-4d2f-899b-dcec3841ac16.jpg)

6. Mandiri
![mandiri](https://user-images.githubusercontent.com/88444330/188271763-0a8b11af-5228-4122-8cf0-64942c2a8190.jpg)
8. TK
![TK](https://user-images.githubusercontent.com/88444330/188271762-a3e040f2-1203-4750-9558-e16f0171f781.jpg)
9. After TK (show score)
![TKscore](https://user-images.githubusercontent.com/88444330/188271761-90d93b83-05fe-46a8-9f23-c0bcdee3476c.jpg)
10. feedback
-->![feedback](https://user-images.githubusercontent.com/88444330/188271759-5b1394df-7a75-4d9f-b1a3-f4540ba7beec.jpg)

Reply:
> A bit forget about the frontend of practicum sequence, can you check later? @adhiiimf

sorry for late respond, okey bang i'll check this

Reply:
> For the frontend of practicum sequence is:
> 
> 1. Intro
>    ![start](https://user-images.githubusercontent.com/88444330/188271757-a53a5cd3-fb99-492f-9b1e-7edf31c5c006.jpg)
> 2. TA start
>    ![TA](https://user-images.githubusercontent.com/88444330/188271767-f50839ee-ba8a-4b1c-9a06-a2959d3ba4dc.jpg)
> 3. After TA (show the score)
>    ![score TA](https://user-images.githubusercontent.com/88444330/188271765-2385893a-dcaf-4ccf-bb55-33e903b504cf.jpg)
> 4. Jurnal
>    ![jurnal](https://user-images.githubusercontent.com/88444330/188271764-2f6afaa7-eb27-4d2f-899b-dcec3841ac16.jpg)
> 5. Mandiri
>    ![mandiri](https://user-images.githubusercontent.com/88444330/188271763-0a8b11af-5228-4122-8cf0-64942c2a8190.jpg)
> 6. TK
>    ![TK](https://user-images.githubusercontent.com/88444330/188271762-a3e040f2-1203-4750-9558-e16f0171f781.jpg)
> 7. After TK (show score)
>    ![TKscore](https://user-images.githubusercontent.com/88444330/188271761-90d93b83-05fe-46a8-9f23-c0bcdee3476c.jpg)
> 8. feedback
>    -->![feedback](https://user-images.githubusercontent.com/88444330/188271759-5b1394df-7a75-4d9f-b1a3-f4540ba7beec.jpg)

okayy good, that's right wwkwk good job dip